tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_internal_mode.py

```
@@ -0,0 +1,306 @@
+
+logger = logging.getLogger(__name__)
+
+
+def skip_if_provider_mode():
+    """
+    Skip test if running on a provider cluster (with consumers).
+
+    Internal mode tests should only run on standalone/internal clusters.
+    """
+    if getattr(config, "multicluster", False):
+        if config.is_consumer_exist() or config.hci_client_exist():
```

Ruff (0.15.21)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)